core/schema/service.go

```diff
@@ -193,6 +194,14 @@ func (s *Service) identifySchemaChange(ctx context.Context, request *changedetec
 		return fmt.Errorf("got error while identifying schema change for namespace : %s, schema: %s, version: %d, %s", request.NamespaceID, request.SchemaName, request.Version, err)
 	}
 	log.Printf("schema change result %s", sce.String())
+	return s.sendNotification(ctx, sce, request)
+}
+
+func (s *Service) sendNotification(ctx context.Context, sce *stencilv1beta1.SchemaChangedEvent, request *changedetector.ChangeRequest) error {
+	schemaID, err := s.repo.GetSchemaID(ctx, request.NamespaceID, request.SchemaName)
```

Comment:
pass SchemaID in parameters if its empty then call db to fetch it as from above method we already have schemaID only with API call we will not have schemaID.